cate/util/misc.py

```diff
@@ -30,12 +30,44 @@
 from datetime import datetime, date, timedelta
 from io import StringIO
 from typing import Union, Tuple, Sequence, Optional, Iterable
-
+from tornado import httpclient
 import numpy as np
 
 __author__ = "Norman Fomferra (Brockmann Consult GmbH)"
 
 
+def checkUrl(url: str):
+    """
+    This method check wehnever a strng is an url or not
+    it also validate the url, this mean that exists an active
+    remote service and it reply to a get request.
+    In case of failure the error is reported
+    Value error mean in this case that is not an url, may be a file
+    :param url: url string or file path to check
+    :return
+    """
+    http_client = httpclient.HTTPClient()
+    report = {'is_url': True, 'is_valid': True, 'error': None}
```

Comment:
Be more explicit, you can use 

    report = dict(is_url=True, is_valid=True, error=None)